File: Lista 2 sala/C/atividade02.c
#include <stdio.h>

int main() {
    float nota1, nota2, nota3, resultado;

    printf("Qual a sua nota 1? ");
    scanf("%f", &nota1);

    printf("Qual a sua nota 2? ");
    scanf("%f", &nota2);

    printf("Qual a sua nota 3? ");
    scanf("%f", &nota3);

    resultado = (nota1 + nota2 + nota3) / 3;

    if (resultado >= 6.0) {
        printf("Aluno aprovado. Media: %.2f\n", resultado);
    } else {
        printf("Aluno nao aprovado. Media: %.2f\n", resultado);
    }

    return 0;
}

File: Lista 2 sala/Python/atividade02.py
nota1 = float(input("Digite sua primeira nota: "))
nota2 = float(input("Digite sua segunda nota: "))

media = (nota1 + nota2) / 2

if media >= 6.0:
    print(f"Parabéns, você foi aprovado! Sua média é {media:.2f}.")
else:
    print("Você está de recuperação.")
    exame = float(input("Digite o valor do seu exame: "))
    
    nova_media = (media + exame) / 2
    
    if nova_media >= 5.0:
        print(f"Você foi aprovado em exame. Sua nova média é {nova_media:.2f}.")
    else:
        print(f"Você não foi aprovado. Sua nova média é {nova_media:.2f}.")
